packages/traceability/issuer-irec-api/src/pods/certification-request/handlers/create-irec-certification-request.handler.ts

```diff
@@ -5,17 +5,17 @@ import { createReadStream } from 'fs';
 import {
     BlockchainPropertiesService,
     CertificationRequest,
-    CreateCertificationRequestHandler as OriginalHandler
+    CreateCertificationRequestCommand
 } from '@energyweb/issuer-api';
 import { FileService, UserService } from '@energyweb/origin-backend';
+
 import { CreateIrecCertificationRequestCommand } from '../commands';
 import { IrecCertificateService } from '../irec-certificate.service';
 import { FullCertificationRequestDTO } from '../full-certification-request.dto';
 import { IrecCertificationRequest } from '../irec-certification-request.entity';
 
 @CommandHandler(CreateIrecCertificationRequestCommand)
-export class CreateCertificationRequestHandler
-    extends OriginalHandler
+export class CreateIrecCertificationRequestHandler
```

Comment:
Why did you change the name?

Reply:
It probably wont work with `origin-irec-backend`

Reply:
Changed the name because it works differently now - it calls the original handler and supplements it with I-REC features. It doesn't override the original handler.

Why won't it work? I'll do more thorough testing on this.

Reply:
Manually only, I'll add tests later

Reply:
added tests